frontend/src/pages/modelServing/screens/projects/ProjectRuntimeMetricsWrapper.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+import MetricsPage from '~/pages/modelServing/screens/metrics/MetricsPage';
+import { ModelServingMetricsProvider } from '~/pages/modelServing/screens/metrics/ModelServingMetricsContext';
+import { getRuntimeMetricsQueries } from '~/pages/modelServing/screens/metrics/utils';
+import NotFound from '~/pages/NotFound';
+import { ProjectDetailsContext } from '~/pages/projects/ProjectDetailsContext';
+import { getProjectDisplayName } from '~/pages/projects/utils';
+import RuntimeGraphs from '~/pages/modelServing/screens/metrics/RuntimeGraphs';
+import { MetricType } from '~/pages/modelServing/screens/types';
+
+const ProjectInferenceMetricsWrapper: React.FC = () => {
+  const {
+    currentProject,
+    servingRuntimes: { data: runtimes, loaded },
+  } = React.useContext(ProjectDetailsContext);
+  const runtime = runtimes[0];
+  if (!loaded) {
+    return (
+      <Bullseye>
+        <Spinner />
+      </Bullseye>
+    );
+  }
+  if (!runtime) {
+    return <NotFound />;
+  }
+  const queries = getRuntimeMetricsQueries(runtime);
+  const projectDisplayName = getProjectDisplayName(currentProject);
+
+  return (
+    <ModelServingMetricsProvider queries={queries} type={MetricType.RUNTIME}>
+      <MetricsPage
+        title={`ovm metrics`}
```

Comment:
This is a fixed label but we are currently supporting custom runtimes, it might be weird having a different runtime and displaying ovms. And I think is ovms (OpenVINO Model Server).

Reply:
I understand it's fixed, this will need to be adjusted when we get the runtime proper changes... 

cc @vconzola please address the comment about the name -- I was given this "ovm" concept from you. I can definitely expand it -- I just didn't know what OVM was 😛

Reply:
"ovms" is what shows up in the model server table Type column. I'm not sure where that text comes from because the user currently doesn't select a runtime. It must come the backend someplace. I want what's in the chart title to match what's in the table so the user can make a 1-1 connection. Just FYI, once we support multiple servers (which should be next sprint, I think) the "Type" is going to be replace by a "Name", so all this will change. I'll show what I mean in the UX meeting tomorrow.